src/Account.ts

```diff
@@ -279,147 +120,20 @@ export class Account {
-   * @param params Contains key vault which indicates the contract this function should interact with
-   * @returns A boolean indicating of fees were collected or not
-   */
-  async settleInterest(params: { vault: Vault }): Promise<boolean> {
-    await this.initialiseReachAccount();
-    const ctc = this.reachAccount.contract(backend, params.vault.id);
-    const put = ctc.a.FeeCollector;
-    const res = await put.settleInterest();
-    return res;
-  }
-
-  /**
-   * Will trigger interest to accrue on a specific user vault
-   * @param params Contains address of vault to accrue interest for. Also includes vault which indicates the contract this function should interact with.
-   * @returns A boolean indicating of fees were collected or not
-   */
-  async dripInterest(params: { address: string; vault: Vault }): Promise<boolean> {
-    await this.initialiseReachAccount();
-    const ctc = this.reachAccount.contract(backend, params.vault.id);
-    const put = ctc.a.FeeCollector;
-    const res = await put.dripInterest(params.address);
-    return res;
-  }
-
   /**
    *
-   * @param params Contains key address which indicates the vault we want to retrieve the info of as well as key vault that indicates the contract we want to interact with
-   * @returns the information for the specified vault
-   */
-
-  async getUserInfo(params: { address: string; vault: Vault }): Promise<UserVaultReturnParams> {
-    await this.initialiseReachAccount();
-
-    const userVault = await params.vault.getUserInfo({ account: this, address: params.address });
-    // NOTE: does not account for leap year
-    const vaultState = await this.getVaultState({ vault: params.vault });
-    const VAULT_INTEREST_RATE = vaultState.interestRate;
-    const now = await this.reachStdLib.getNetworkSecs();
-    const interestAccrued = calculateInterestAccrued(
-      now.toNumber(),
-      userVault.lastAccruedInterestTime,
-      userVault.vaultDebt,
-      VAULT_INTEREST_RATE,
-    );
-    userVault.vaultDebt += interestAccrued;
-    return userVault;
-  }
-
-  /**
-   * Returns the contract address
-   * @param params An object with key vault that indicates the contract whose address is to be retrieved
-   * @returns A formatted address of the specified contract as a string
-   */
-  async getContractAddress(params: { vaultId: number }): Promise<string> {
-    await this.initialiseReachAccount();
-    const ctc = this.reachAccount.contract(backend, params.vaultId);
-    const contractAddress = await ctc.getContractAddress();
-    return this.reachStdLib.formatAddress(contractAddress);
-  }
-
-  /**
-   * 
-   * @param params vaultId which indicates the contract we want to interact with
-   */
-  async getContractAbi(params: { vaultId: number }): Promise<AbiInterface> {
-    await this.initialiseReachAccount();
-    const ctc = this.reachAccount.contract(backend, params.vaultId);
-    return await ctc.getABI();
-  }
```

Comment:
@AwolDes 
I think this function can be modified to receive the backend as a parameter so the `VaultClient` and `StakingClient` can both share this